tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -93,13 +121,38 @@ func ConfigInit(ip string) (string, error) {
 	}
 
 	log.Printf("Initializing the SingleNode Config DB on esx [%s] \n", ip)
-	return ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+	out, err := ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+
+	if err != nil {
+		return out, err
+	}
+
+	// verify the DB init
+	if GetDBmode(ip) != admincli.DBSingleNode {
```

Comment:
No specific reason. Just happened to write the if condition in that way. Would you want it to be changed as per style in CreateVmGroup and DeleteVmGroup?